android/cli/hooks/aar-transform.js

```diff
@@ -45,6 +46,7 @@ exports.init = function (logger, config, cli) {
 	cli.on('build.pre.compile', {
 		post: function(builder, callback) {
 			scanProjectAndStartTransform(builder, logger, callback);
+			registerHyperloopCompatibilityFixes(cli, builder, logger);
```

Comment:
`scanProjectAndStartTransform()` is async. The call to `registerHyperloopCompatibilityFixes()` must be done in `scanProjectAndStartTransform()`'s callback or if there's no conflict, `registerHyperloopCompatibilityFixes()` could simply be run before calling `scanProjectAndStartTransform()`.

```js
scanProjectAndStartTransform(builder, logger, function (err) {
    if (!err) {
        registerHyperloopCompatibilityFixes(cli, builder, logger);
    }
    callback(err);
});
```